cico_pr_test.sh

```diff
@@ -11,15 +11,15 @@ delete_tmp() {
     rm -rf $NEW_JOBS $MASTER_JOBS
 }
 
-jenkins-jobs test $JJB_INDEX -o $NEW_JOBS
+/home/devtools/venv/env/bin/jenkins-jobs test $JJB_INDEX -o $NEW_JOBS
```

Comment:
Perhaps `~/venv/env/bin/jenkins-jobs` so it doesn't depend on the user.

Reply:
good catch! will update